CHANGELOG.md

```diff
@@ -42,6 +42,8 @@
 - Updated the bundled libjpeg, libpng, Freetype, and SDL2 libaries for macOS to not use AVX instructions, since they are incompatible with the CPUs in some older Mac models. This fixes an issue with SIGILLs in native builds.
 
 ### Native
+## ScrollView performance 
+- Improved perceived ScrollView performance by preventing caching while pointers are pressed on them, avoiding inconsistent framerates.
```

Comment:
This is *still* a bad merge. Please actually fix it. This change should go in the top of the changelog, not down in the old part of some previous release.